css/base.css

```diff
@@ -255,12 +255,19 @@ input[type='text'] {
 .tweet .text {
   clear: right;
   word-wrap: break-word;
+  white-space: pre-wrap;
```

Comment:
Might also be needed for your `<br>` padding of the picture; I'm going to change that to 0.5em margins.